libs/sdk/src/common/push-protocol.ts

```diff
@@ -19,6 +36,8 @@ class PushProtocol {
   private _pushClient?: PushAPI;
   private _notificationsStream?: PushStream;
   private _akashaChannelAccount = '0x7Ebe714066149bEef3c2f60cC89eadF50Da23C26';
+  private _unreadNotifications: PushOrgNotification[] = [];
+  public readonly lastestSeenSidKey = 'lastestSeenSidKey';
```

Comment:
this needs to be prefixed with the current authenticated did

Reply:
added a method that prepends it